gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/longrunning/stub/HttpJsonOperationsStub.java

```diff
@@ -249,38 +293,30 @@ protected HttpJsonOperationsStub(
       OperationsStubSettings settings,
       ClientContext clientContext,
       HttpJsonStubCallableFactory callableFactory,
-      TypeRegistry typeRegistry)
+      TypeRegistry typeRegistry,
```

Comment:
Same as above, adding parameters to protected methods are breaking changes.

Reply:
I'll try to overload instead

Reply:
Can this constructor be private?

Reply:
Ah yeah, I believe so. I'll make it private.